Sources/DangerSwiftPeripherySampleApp/DangerSwiftPeripherySampleApp.swift

```diff
@@ -2,6 +2,8 @@ public struct DangerSwiftPeripherySampleApp {
     public private(set) var text = "Hello, World!"
```

Comment:
<!--
  0 failure: 
  1 warning:  Property 'text' i...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerSwiftPeripherySampleApp/DangerSwiftPeripherySampleApp.swift;
  Line: 2;
-->

- :warning: Property 'text' is declared public, but not used outside of DangerSwiftPeripherySampleApp